src/contracts/Staking.sol

```diff
@@ -461,9 +461,13 @@ contract Staking is OwnableUpgradeable, StakingStorage {
         @notice retrieve reward tokens from warmup
         @dev if user has funds in warmup then user is able to claim them (including rewards)
         @param _recipient address
+        @param _trigger bool - should trigger a rebase
      */
-    function claim(address _recipient) public {
+    function claim(address _recipient, bool _trigger) public {
```

Comment:
would opt for a more descriptive var name, `_triggerRebase` or the like

Reply:
I can change all instances of trigger to this